extensions/broker-server/README.md

```diff
@@ -16,7 +16,7 @@
 
 ## About this Extension
 
-Implementation of an IDS Broker Server as an EDC Extension.
+Implementation of an Broker Server as an EDC Extension.
```

Comment:
grammar: "of a Broker"

will also suggest rewording: "EDC Broker backend as an EDC Extension."

Reply:
fixed